src/main/java/uk/gov/justice/digital/services/RedShiftTableExpiryService.java

```diff
@@ -21,7 +21,7 @@ public class RedShiftTableExpiryService {
             "FROM SVV_EXTERNAL_TABLES " +
             "WHERE schemaname = 'reports' " +
             "AND COALESCE(json_extract_path_text(parameters, 'transient_lastDdlTime', TRUE), '0')::bigint < (EXTRACT(EPOCH FROM GETDATE()) - %d)";
-
+    private static final int DROP_BATCH_SIZE = 500;
```

Comment:
Out of curiosity do we know what the limit is the longest query?

Reply:
Is it 100K characters as you mentioned for each statement that gets executed?

Reply:
Yes, that's right. And each SQL statement is 68 bytes long. It's currently failing to remove 1850 tables in dev.